ui/pages/DataManagement/components/TriggerSearchDataUpdatePages.jsx

```diff
@@ -46,25 +46,25 @@ const FAMILY_FIELDS = [
   DATASET_TYPE_FIELD,
 ]
 
-const TriggerSearchDataUpdateForm = ({ path, fields }) => (
+const TriggerSearchDataUpdateForm = ({ entity, fields }) => (
   <SubmitFormPage
-    header={`Trigger ${snakecaseToTitlecase(path)}`}
-    url={`/api/data_management/trigger_${path}`}
+    header={`Trigger Delete ${snakecaseToTitlecase(entity)} Search`}
```

Comment:
I thought "Trigger Delete Project Search" was a better message for what happens with this action than the current "Trigger Delete Project"